test/v1/connectors/AmendDisclosuresConnectorSpec.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import mocks.MockAppConfig
+import uk.gov.hmrc.domain.Nino
+import v1.mocks.MockHttpClient
+import v1.models.domain.DesTaxYear
+import v1.models.outcomes.ResponseWrapper
+import v1.models.request.disclosures.{AmendDisclosuresRequest, AmendDisclosuresRequestBody, AmendTaxAvoidance}
+
+import scala.concurrent.Future
+
+class AmendDisclosuresConnectorSpec extends ConnectorSpec {
+
+  private val nino: String = "AA111111A"
+  private val taxYear: String = "2019"
+
+  val amendTaxAvoidance: AmendTaxAvoidance = AmendTaxAvoidance("12345","2020-12")
```

Comment:
Use the right values i.e. 14211123 and 2020-21

Reply:
Sure. Made that change for the test data.